src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -100,7 +113,16 @@ public void start() {
       addBombardmentSources();
       m_needToAddBombardmentSources = false;
     }
+    fightCurrentBattle();
+    if (rocketHelper != null && !games.strategy.triplea.Properties.getRollRocketsAfterSBR(getData())) {
+      rocketHelper.fireRockets(m_bridge, m_bridge.getPlayerID());
+      rocketHelper = null;
```

Comment:
Can you update RocketHelper and/or the usage so we do not have to re-initialize it, and also so we are not dependent upon it being null. IE: please fix the SRP violation for rocket helper.